### DISCUSSION TITLE
[Bug]: Selected object position change when use fabric.ActiveSelection()

### DISCUSSION THREAD

**Discussion by niketaOptimumnbrew** [fabricjs/fabric.js#9768] [Q&A]
### CheckList

- [X] I agree to follow this project's Code of Conduct
- [X] I have read and followed the [Contributing Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md)
- [X] I have read and followed the [Issue Tracker Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md#%EF%B8%8F-issue-tracker)
- [X] I have searched and referenced existing issues and discussions
- [X] I am filing a **BUG** report.
- [X] I have managed to reproduce the bug **after** upgrading to the **latest** version
- [X] I have created an accurate and minimal reproduction

### Version

5.3.0

### In What environments are you experiencing the problem?

Firefox, Chrome, Safari

### Node Version (if applicable)

14.20.0

### Link To Reproduction

https://jsfiddle.net/Niketa_patel/p4s8n1z7/2/

### Steps To Reproduce

1. Add 2 object on to the canvas.
2. select both object.
3.you can see that selection of both object move to the corner of canvas and also it's position change.
4. for more detail you can check above link in jsfiddle to generate the error.

### Expected Behavior

- Position of selection of selected objects should not change unless and until update by manually.

### Actual Behavior

- Position of selection of selected objects change and also selection moved to the corner of canvas.
- Please check below screen short
[
![image](https://github.com/fabricjs/fabric.js/assets/126561906/583bc876-5abe-407e-8328-1019f1294d6c)
](url)
[Uploading screen-capture (14).webm…]()

**Comment by ShaMan123** [comment]
seems to me you are misusing the selection created event and nesting an active selection inside an active selection

**Reply by niketaOptimumnbrew** [reply]
@ShaMan123 How to set left and top position when their multiple object selected?
In selection:created event it's giving me separated object not a common object.
event if I can't use getActiveGroup(); method cause of the version update.

Can you please help me for that?
It would be really appreciated :)

**Reply by ShaMan123** [reply]
I don't understand what you want to do.
Under v6 it is too hard to mess around with groups/selection

**Reply by niketaOptimumnbrew** [reply]
@ShaMan123 Ok Let me clear the question for you.
I want one single object which contain left and right property when there are more than one objects selected.

**Comment by asturur** [comment]
Is completely unclear what is the scope of your selection event @niketaOptimumnbrew.
What do you want to do?
In the moment a selection is created, if there are more than one object selected you already have an active selection, why do you create another?

**Reply by niketaOptimumnbrew** [reply]
@asturur Thank you for the clarification regarding the ActiveSelection event. However, I have a follow-up question: If multiple objects are selected, could you guide me on why their are not top and left property? cause in latest version in selected:created event not getting top/left property same as in old version.
I'm moving multiple selected object position according to the top/left property to the whole object when there are more then one objects.

Please check below screen short:(version 1.7.22)

![image](https://github.com/fabricjs/fabric.js/assets/126561906/199178f6-50dc-41b0-84c2-2c8990815efb)

I want this type of target object so I can manage the position.

Please check below screen short(version 5.3.0)

![image](https://github.com/fabricjs/fabric.js/assets/126561906/98db6048-65d9-4ff1-b1ff-0accc3caa701)

Please check  above screen short ,  getting only two object not getting top/left property in selected object.

Please refer this issue: #9769 to check both version json.

**Comment by guda-art** [comment]
How to get the absolute position of a single object in a group relative to the canvas

**Reply by asturur** [reply]
This question has been asked many times, please use a bit of effort in searches
